### DISCUSSION TITLE
Continuous dimension string

### DISCUSSION THREAD

**Discussion by theoryshaw** [mozman/ezdxf#700] [Q&A] [answered]
Is there a way to do something like the following with ezdxf?  Or is this not supported with the dxf standard?

![2022-07-04_16-14-45_Autodesk_Revit_2020 2_-_ Project1_-_Floor_Plan_Le_Revit](https://user-images.githubusercontent.com/507113/177217212-206d0e40-148d-4813-acca-d14a66fd030a.gif)

**Comment by mozman** [comment, marked as answer]
Tutorial for linear dimensions: https://ezdxf.mozman.at/docs/tutorials/linear_dimension.html

Add linear dimension for multiple points: https://ezdxf.mozman.at/docs/layouts/layouts.html#ezdxf.layouts.BaseLayout.add_multi_point_linear_dim

Example: 
https://github.com/mozman/ezdxf/blob/6cdf7099225ca43e0b3baf729446003f66256ab3/examples/render/dimension_linear.py#L511

**Reply by theoryshaw** [reply]
Thanks @mozman 
If curious...we're trying to brainstorm how we can codify dimensioning into the IFC standard. 
https://community.osarch.org/discussion/1014/ifc-an-approach-for-dimension-line-offsets#latest
